server/src/org/apache/cloudstack/region/RegionManagerImpl.java

```diff
@@ -229,7 +232,7 @@ public boolean deleteUser(DeleteUserCmd cmd) {
      */
     @Override
     public Domain updateDomain(UpdateDomainCmd cmd) {
-        return _domainMgr.updateDomain(cmd);
+        return _managementService.updateDomain(cmd);
```

Comment:
@yvsubhash : As per your suggestion, I have changed the search query and also removed the unused method updateDomain from managementserviceimpl class.

Please check.